samples/wifi/raw_tx_packet/Kconfig

```diff
@@ -12,6 +12,10 @@ config CONNECTION_TIMEOUT
 	int "Time to wait for a station to connect"
 	default 30
 
+config DHCP_TIMEOUT_S
```

Comment:
Any reason this Kconfig and the one above not having the "sample" prefix?

Reply:
There is no specific reason for not having the "sample" prefix in those particular Kconfig options.
Added now.